tests/tests/tier0/bluechi-agent-config-c-option/config-files-agent/valid.conf

```
@@ -0,0 +1,61 @@
+# This is the configuration file for bluechi-agent.
+#
+# All available options are described in bluechi-agent.conf man page or in
+# agent.conf file provided by bluechi-agent package.
+#
+
+[bluechi-agent]
+#
+# The unique name of this agent.
+# NodeName defaults the system's hostname.
+NodeName="node-bar"
+
+#
+# The IP address that bluechi-agent can use to connect to bluechi manager. It must be a valid IPv4 or IPv6 address.
+# One of the ManagerHost or ManagerAddress options must be set for each bluechi-agent.
+ManagerHost=192.168.1.56
```

```suggestion
#ManagerHost=192.168.1.56
```
Don't set the `ManagerHost` here. It is determined during test execution and should not be overwritten. This also causes it to fail, currently. 
In more detail:
The agent tries to connect to this host, which probably does not exist on other envs like the CI here. This results in `systemctl stop bluechi-agent` to wait till any timeout will be triggered - in this case the pytest timeout.